src/service/wallet.tsx

```diff
@@ -103,7 +104,7 @@ export const makeWalletService = () => {
           showToast('Network not found.');
           break;
         case Errors.noSmartWallet:
-          showToast(SmartWalletNotFoundToast);
+          showToast(<SmartWalletNotFoundToast />);
```

Comment:
it's `showToast`'s job to render. It should be able to take a component without JSX. i.e.,
```
  const showToast = (component: ReactComponent, options?: ToastOptions) => {
    const content: ToastContent = React.createElement(component, null);
    toastId = toast.error(content, options);
  };
```
Then this file can stay `.ts` and this line needn't change.

Reply:
Very nice suggestion, done. However, I'm leaving the interface for `showToast` unchanged because `ToastContent` can also be a string, not always a component. I think of this as a small wrapper on `toast.error` just to persist the toast id.